src/apis/StyleSheet.res

```diff
@@ -7,5 +7,3 @@ external absoluteFillObject: Style.t = "absoluteFillObject"
 
 @module("react-native") @scope("StyleSheet")
 external create: Js.t<'a> => Js.t<'a> = "create"
-@module("react-native") @scope("StyleSheet")
-external flatten: array<Style.t> => Style.t = "flatten"
```

Comment:
I am wondering if this can still work per RN docs. It's supposed to handle exactly this: flatten an array. Since we type styles != type t, this could still work right ?

Reply:
Ah, I think you are right, I was confusing it with "%identity", this should still work as it is. So I can add it back.

But you won't be able to directly pass the result directly to a view's style prop anymore (as that expects an array now).